doc/ir-semantics.text

```diff
@@ -108,18 +118,30 @@ Update (partial) function, returns a S:
        = step CC[v] (P, T, H, E*, C*)
     when v := eval P H E e
 
+  step   CC[call f (v, ...)]
+         (P, T, H, E*, C*)
+       = (P, T, H, E* E', C* C)
```

Comment:
Can't we store the current lexical environment away (to reinstate it on `return`)?

Also `C* C`, you mean `C* CC`? (Or conversely `CC[call ..]` -> `C[call ...])